### PR TITLE
relnote(118): CSS math functions enabled (CSS Values 4)

### PR DESCRIPTION
**Pull request by bsmth**
This PR adds a relnote about the enabled math functions and brings them out of experimental status. There needs to be related Yari (sidebar) and BCD PRs to indicate that these are out of experimental status.

Moving the 112 relnote into 118 because this was nightly-only until https://hg.mozilla.org/integration/autoland/rev/d84e836e47652db87a27a6f4b689c2a34c699685

__Related issues and pull requests:__
- [ ] Parent issue https://github.com/mdn/content/issues/28840
- [x] BCD https://github.com/mdn/browser-compat-data/pull/20652
- [x] `layout.css.exp.enabled`: https://github.com/mdn/content/pull/26203
  - `pow()`, `sqrt()`, `hypot()`, `log()` and `exp()`  mentioned in 112 was nightly-only until now

__Bugzilla:__
- https://bugzilla.mozilla.org/show_bug.cgi?id=1814589

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/118/index.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ This article provides information about the changes in Firefox 118 that affect d
 
 ### CSS
 
+- Multiple CSS [math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) are now supported.
```
Comment (chrisdavidmills) [inline files/en-us/mozilla/firefox/releases/118/index.md:21]: The phrasing of this bit seems a little strange. How about:

> Multiple CSS [math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) are now supported: [`abs()`](/en-US/docs/Web/CSS/abs), [`sign()`](/en-US/docs/Web/CSS/sign), [`round()`](/en-US/docs/Web/CSS/round), [`mod()`](/en-US/docs/Web/CSS/mod), [`rem()`](/en-US/docs/Web/CSS/rem), [`pow()`](/en-US/docs/Web/CSS/pow), [`sqrt()`](/en-US/docs/Web/CSS/sqrt), [`hypot()`](/en-US/docs/Web/CSS/hypot), [`log()`](/en-US/docs/Web/CSS/log), and [`exp()`](/en-US/docs/Web/CSS/exp) (Firefox bug [1814589](https://bugzil.la/1814589)).

**Review by chrisdavidmills — APPROVED** [review]
LGTM @bsmth, pending a slightly nitpicky quibble about one bit of phrasing.

File: files/en-us/mozilla/firefox/releases/118/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/118/index.md]: Yeah, I agree. Edited in da65f7878b75138f773bdb871a47b05af189290f

**Comment by bsmth** [comment]
Thank you @chrisdavidmills 🙏🏻

**Pull request merged**